docs/docs/layout-animations/layout-animation-config.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 5
+---
+
+# Skipping Layout Animations
+
+`LayoutAnimationConfig` is a wrapper that lets you skip entering and exiting animations.
+
+## Reference
+
+```javascript
+import Animated, {
+  LayoutAnimationConfig,
+  PinwheelIn,
+  PinwheelOut,
+} from 'react-native-reanimated';
```

Comment:
```suggestion
import { LayoutAnimationConfig } from 'react-native-reanimated';
```

This is debatable but everywhere else in the documentation I've only imported the function on which the documentation says about at that page eg. https://docs.swmansion.com/react-native-reanimated/docs/core/useAnimatedStyle#reference

Full working example is usually in the `Example` section. The reference is just the gist about the feature.

Again, that's debatable. I'm okay with having all the imports but we'd have to make it consistent across all pages.